### ISSUE TITLE
vt_teature_info branch - Leaflet does not align splitter correctly until window is resized

### ISSUE THREAD

**Issue by a-stacey** [TerriaJS/terriajs#2729]
Steps to reproduce:
Instance 1)
1) Start terria.
2) Switch to 2D.
3) Close terria.
4) Open terria.
5) Add layer to terria, split layer, turn off layer on one side of the split.
6) Move the splitter position

Use this share link:
1) http://localhost:3001/#start=%7B%22version%22%3A%220.0.05%22%2C%22initSources%22%3A%5B%7B%22initFragment%22%3A%22aremi%22%7D%2C%22proxy%2F_60s%2Fhttp%3A%2F%2Fstatic.aremi.nicta.com.au%2Finit.json%22%2C%7B%22catalog%22%3A%5B%7B%22name%22%3A%22User-Added+Data%22%2C%22description%22%3A%22The+group+for+data+that+was+added+by+the+user+via+the+Add+Data+panel.%22%2C%22info%22%3A%5B%5D%2C%22isUserSupplied%22%3Atrue%2C%22isPromoted%22%3Afalse%2C%22isHidden%22%3Afalse%2C%22forceProxy%22%3Afalse%2C%22customProperties%22%3A%7B%7D%2C%22id%22%3A%22Root+Group%2FUser-Added+Data%22%2C%22shortReportSections%22%3A%5B%5D%2C%22isWaitingForDisclaimer%22%3Afalse%2C%22hideSource%22%3Afalse%2C%22nameInCatalog%22%3A%22User-Added+Data%22%2C%22isOpen%22%3Atrue%2C%22items%22%3A%5B%7B%22name%22%3A%22Solar+Satellite+DNI+%26+GHI+%28copy%29%22%2C%22isUserSupplied%22%3Atrue%2C%22isPromoted%22%3Afalse%2C%22isHidden%22%3Afalse%2C%22forceProxy%22%3Afalse%2C%22customProperties%22%3A%7B%7D%2C%22id%22%3A%22Root+Group%2FUser-Added+Data%2FSolar+Satellite+DNI+%26+GHI+%28copy%29%22%2C%22shortReportSections%22%3A%5B%5D%2C%22isLoading%22%3Afalse%2C%22isWaitingForDisclaimer%22%3Afalse%2C%22hideSource%22%3Afalse%2C%22nameInCatalog%22%3A%22Solar+Satellite+DNI+%26+GHI%22%2C%22nowViewingIndex%22%3A0%2C%22rectangle%22%3A%5B112%2C-48.00000000000001%2C155%2C-5%5D%2C%22url%22%3A%22http%3A%2F%2Fgis.aremi.nationalmap.gov.au%2Fbom%2Fwms%22%2C%22dataCustodian%22%3A%22The+Bureau+of+Meteorology+is+the+data+custodian+of+the+source+Australian+Hourly+Solar+Irradiance+Gridded+Data.%3Cp%3EThe+Bureau+of+Meteorology+provides+Australians+with+environmental+intelligence+for+safety%2C+sustainability%2C+well-being+and+prosperity.+It+aims+to+promote+informed+safety%2C+security+and+economic+decisions+by+governments%2C+industry%2C+and+the+community+through+the+provision+of+information%2C+forecasts%2C+services+and+research+relating+to+weather%2C+climate+and+water.%22%2C%22isEnabled%22%3Atrue%2C%22isShown%22%3Afalse%2C%22isLegendVisible%22%3Atrue%2C%22displayChoicesBeforeLegend%22%3Afalse%2C%22isEnableable%22%3Atrue%2C%22isMappable%22%3Atrue%2C%22showsInfo%22%3Atrue%2C%22featureInfoTemplate%22%3A%7B%22name%22%3A%22%7B%7BGRAY_INDEX%7D%7D+W%2Fm2%22%2C%22template%22%3A%22%3Ch3%3EDNI%3A+%3Cstrong%3E%7B%7BGRAY_INDEX%7D%7D+W%2Fm%26%23178%3B%3C%2Fh3%3E%3Cchart++++title%3D%27DNI+%28%7B%7B%23terria.formatNumber%7D%7D%7BmaximumFractionDigits%3A3%7D%7B%7Bterria.coords.latitude%7D%7D%7B%7B%2Fterria.formatNumber%7D%7D%2C%7B%7B%23terria.formatNumber%7D%7D%7BmaximumFractionDigits%3A3%7D%7B%7Bterria.coords.longitude%7D%7D%7B%7B%2Fterria.formatNumber%7D%7D%29%27++++sources%3D%27http%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FDNI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%3Fstart%3D2014-01-01T00%3A00%3A00%252B1000%26end%3D2014-01-07T23%3A00%3A00%252B1000%2Chttp%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FDNI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%3Fstart%3D2014-01-01T00%3A00%3A00%252B1000%26end%3D2014-12-31T23%3A00%3A00%252B1000%27++++source-names%3D%271-7+Jan+2014%2C2014%27++++downloads%3D%27http%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FDNI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%3Fstart%3D2014-01-01T00%3A00%3A00%252B1000%26end%3D2014-12-31T23%3A00%3A00%252B1000%2Chttp%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FDNI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%27++++download-names%3D%272014%2C+1990-2015%27++++preview-x-label%3D%27DNI+sample%27++++column-names%3D%27UTC+Time%2CDNI%27++++column-units%3D%27Date%2CW%2Fm%26%23178%3B%27%3E%3C%2Fchart%3E%3Cp%3EDownload+25+year+DNI+hourly+timeseries+for+this+point%3A+%3Cstrong%3E%3Ca+href%3D%27http%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FDNI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%27%3E1990+-+2015+%28CSV%29%3C%2Fa%3E%3C%2Fstrong%3E%3C%2Fp%3E%3Ch3%3EGHI%3C%2Fh3%3E%3Cchart++++title%3D%27GHI+%28%7B%7B%23terria.formatNumber%7D%7D%7BmaximumFractionDigits%3A3%7D%7B%7Bterria.coords.latitude%7D%7D%7B%7B%2Fterria.formatNumber%7D%7D%2C%7B%7B%23terria.formatNumber%7D%7D%7BmaximumFractionDigits%3A3%7D%7B%7Bterria.coords.longitude%7D%7D%7B%7B%2Fterria.formatNumber%7D%7D%29%27++++sources%3D%27http%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FGHI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%3Fstart%3D2014-01-01T00%3A00%3A00%252B1000%26end%3D2014-01-07T23%3A00%3A00%252B1000%2Chttp%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FGHI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%3Fstart%3D2014-01-01T00%3A00%3A00%252B1000%26end%3D2014-12-31T23%3A00%3A00%252B1000%27++++source-names%3D%271-7+Jan+2014%2C2014%27++++downloads%3D%27http%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FGHI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%3Fstart%3D2014-01-01T00%3A00%3A00%252B1000%26end%3D2014-12-31T23%3A00%3A00%252B1000%2Chttp%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FGHI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%27++++download-names%3D%272014%2C+1990-2015%27++++preview-x-label%3D%27GHI+sample%27++++column-names%3D%27UTC+Time%2CGHI%27++++column-units%3D%27Date%2CW%2Fm%26%23178%3B%27%3E%3C%2Fchart%3E%3Cp%3EDownload+25+year+GHI+hourly+timeseries+for+this+point%3A+%3Cstrong%3E%3Ca+href%3D%27http%3A%2F%2Fservices.aremi.data61.io%2Fsolar-satellite%2Fv1%2FGHI%2F%7B%7Bterria.coords.latitude%7D%7D%2F%7B%7Bterria.coords.longitude%7D%7D%27%3E1990+-+2015+%28CSV%29%3C%2Fa%3E%3C%2Fstrong%3E%3C%2Fp%3E%22%7D%2C%22maximumShownFeatureInfos%22%3A100%2C%22zoomOnEnable%22%3Afalse%2C%22dateFormat%22%3A%7B%7D%2C%22useOwnClock%22%3Atrue%2C%22opacity%22%3A0.6%2C%22treat404AsError%22%3Afalse%2C%22treat403AsError%22%3Atrue%2C%22ignoreUnknownTileErrors%22%3Atrue%2C%22keepOnTop%22%3Afalse%2C%22clipToRectangle%22%3Atrue%2C%22isRequiredForRendering%22%3Afalse%2C%22initialTimeSource%22%3A%222015-01-01T00%3A00%3A00Z%22%2C%22splitDirection%22%3A-1%2C%22layers%22%3A%22Solar_Satellite_DNI_2014%22%2C%22styles%22%3A%22%22%2C%22parameters%22%3A%7B%22srs%22%3A%22EPSG%3A3857%22%7D%2C%22tilingScheme%22%3A%22web-mercator%22%2C%22getFeatureInfoFormats%22%3A%5B%7B%22type%22%3A%22json%22%2C%22format%22%3A%22application%2Fjson%22%7D%2C%7B%22type%22%3A%22xml%22%2C%22format%22%3A%22text%2Fxml%22%7D%5D%2C%22populateIntervalsFromTimeDimension%22%3Atrue%2C%22maxRefreshIntervals%22%3A9000%2C%22disableUserChanges%22%3Afalse%2C%22dataUrl%22%3A%22http%3A%2F%2Fgis.aremi.nationalmap.gov.au%3A80%2Fbom%2Fwcs%3Fservice%3DWCS%26version%3D1.1.1%26request%3DDescribeCoverage%26identifiers%3Dbom%3ASolar_Satellite_DNI_2014%22%2C%22dataUrlType%22%3A%22wcs-complete%22%2C%22type%22%3A%22wms%22%2C%22parents%22%3A%5B%22Root+Group%2FUser-Added+Data%22%5D%7D%5D%2C%22preserveOrder%22%3Afalse%2C%22type%22%3A%22group%22%2C%22parents%22%3A%5B%5D%7D%5D%7D%2C%7B%22sharedCatalogMembers%22%3A%7B%22Root+Group%2FElectricity+Infrastructure%22%3A%7B%22isOpen%22%3Atrue%2C%22type%22%3A%22group%22%2C%22parents%22%3A%5B%5D%7D%2C%22Root+Group%2FRenewable+Energy%22%3A%7B%22isOpen%22%3Atrue%2C%22type%22%3A%22group%22%2C%22parents%22%3A%5B%5D%7D%2C%22Root+Group%2FRenewable+Energy%2FSolar%22%3A%7B%22isOpen%22%3Atrue%2C%22type%22%3A%22group%22%2C%22parents%22%3A%5B%22Root+Group%2FRenewable+Energy%22%5D%7D%2C%2231cf8abc%22%3A%7B%22nowViewingIndex%22%3A1%2C%22isEnabled%22%3Atrue%2C%22isShown%22%3Atrue%2C%22isLegendVisible%22%3Atrue%2C%22useOwnClock%22%3Atrue%2C%22opacity%22%3A0.6%2C%22keepOnTop%22%3Afalse%2C%22initialTimeSource%22%3A%222015-01-01T00%3A00%3A00Z%22%2C%22splitDirection%22%3A1%2C%22styles%22%3A%22%22%2C%22type%22%3A%22wms%22%2C%22parents%22%3A%5B%22Root+Group%2FRenewable+Energy%22%2C%22Root+Group%2FRenewable+Energy%2FSolar%22%5D%7D%2C%22Root+Group%2FUser-Added+Data%22%3A%7B%22isOpen%22%3Atrue%2C%22type%22%3A%22group%22%2C%22parents%22%3A%5B%5D%7D%2C%22Root+Group%2FUser-Added+Data%2FSolar+Satellite+DNI+%26+GHI+%28copy%29%22%3A%7B%22nowViewingIndex%22%3A0%2C%22isEnabled%22%3Atrue%2C%22isShown%22%3Afalse%2C%22isLegendVisible%22%3Atrue%2C%22useOwnClock%22%3Atrue%2C%22opacity%22%3A0.6%2C%22keepOnTop%22%3Afalse%2C%22initialTimeSource%22%3A%222015-01-01T00%3A00%3A00Z%22%2C%22splitDirection%22%3A-1%2C%22styles%22%3A%22%22%2C%22type%22%3A%22wms%22%2C%22parents%22%3A%5B%22Root+Group%2FUser-Added+Data%22%5D%7D%7D%7D%2C%7B%22initialCamera%22%3A%7B%22west%22%3A93.72935790498006%2C%22south%22%3A-37.67307656773083%2C%22east%22%3A-179.71070649321638%2C%22north%22%3A0.5003560087447986%2C%22position%22%3A%7B%22x%22%3A-9528688.820211139%2C%22y%22%3A8882746.911423532%2C%22z%22%3A-5565663.129477033%7D%2C%22direction%22%3A%7B%22x%22%3A0.6718860544980373%2C%22y%22%3A-0.6263394563543689%2C%22z%22%3A0.39530749447079044%7D%2C%22up%22%3A%7B%22x%22%3A-0.2891534767734887%2C%22y%22%3A0.26955200250522665%2C%22z%22%3A0.9185488472668321%7D%7D%2C%22homeCamera%22%3A%7B%22west%22%3A112%2C%22south%22%3A-48.00000000000001%2C%22east%22%3A155%2C%22north%22%3A-5%7D%2C%22baseMapName%22%3A%22Positron+%28Light%29%22%2C%22viewerMode%22%3A%223d%22%2C%22currentTime%22%3A%7B%22dayNumber%22%3A2457023%2C%22secondsOfDay%22%3A43235%7D%2C%22showSplitter%22%3Atrue%2C%22splitPosition%22%3A0.5%7D%5D%7D
2) Move the splitter position

![image](https://user-images.githubusercontent.com/4453477/32425208-8b37a69e-c305-11e7-84fb-7c137507a55e.png)

**Comment by a-stacey** [comment]
Have partially investigated this and this issue is present in both master and vt_feature_info branches and presents against terriajs/aremi-natmap but not in terriajs/TerriaMap.

**Comment by a-stacey** [comment]
Sticking the following debug lines into: lib/Models/Leaflet.js, `function getClipsForSplitter(viewer)`:
```
+ console.log("NW:" + nw);
+ console.log("SE:" + se);
+ console.log("Size:" + size);
+ console.log("Position:" + viewer.terria.splitPosition);
```

Revels that the issue is caused because `const size = map.getSize();` is stale (out of date).

Causing the window to refresh by even a minor manual resize corrects the issue and `const size = map.getSize();` starts returning the correct size.

**Comment by a-stacey** [comment]
This is fixed by #2731
